padle/padle.py

```diff
@@ -0,0 +1,537 @@
+                giveups += 1
+            else:
+                completes += 1
+                average += int(item)
+        guess_monster = await dbcog.find_monster(info[0], ctx.author.id)
+        embed = EmbedView(
+            EmbedMain(
+                title="PADle #{} Stats".format(day)),
+                embed_thumbnail=EmbedThumbnail(
+                    MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        if completes == 0:
+            embed.description = ("**"+MonsterHeader.menu_title(guess_monster).to_markdown()+"**\n" +
+                                 "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: 0%\n"
+                                    "**Average Guess Count**: 0").format(completes, giveups)
+        else:
+            embed.description = ("**"+MonsterHeader.menu_title(guess_monster).to_markdown()+"**" +
+                                "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: {:.2%}\n"
+                                "**Average Guess Count**: {:.2f}").format(completes, giveups, 
+                                                                          completes / (completes + giveups), 
+                                                                          (average / completes))
+        return embed
+    
+    # @padle.command()
+    # @commands.is_owner()
+    # async def fullreset(self, ctx):
+    #     """Resets all stats and information."""
+    #     with open("./pad-cogs/padle/monsters.txt", "r") as f:
+    #         monsters = f.readline().split(",")
+    #         await self.config.padle_today.set(random.choice(monsters))
+
+    #     await self.config.num_days.set(1)
+    #     await self.config.subs.set([])
+    #     await self.config.all_scores.set([])
+    #     await self.config.save_daily_scores.set([])
+    #     all_users = await self.config.all_users()
+    #     for userid in all_users:
+    #         user = await self.bot.fetch_user(userid)
+    #         await self.config.user(user).guesses.set([])
+    #         # need to send message if a user is mid-game
+    #         if(await self.config.user(user).start() and not await self.config.user(user).done()):
+    #             try:
+    #                 await user.send("A full reset occured, the PADle expired.")
+    #             except:
+    #                 pass
+    #         await self.config.user(user).start.set(False)
+    #         await self.config.user(user).done.set(False)
+    #         await self.config.user(user).score.set([])
+    #         await self.config.user(user).edit_id.set("")
+    #         await self.config.user(user).channel_id.set("")
+    #         await self.config.user(user).embed_text.set([])
+    #     await ctx.tick()
+
+    # this takes a long time to run
+    # @padle.command()
+    # async def filter(self, ctx):
+    #     dbcog = await self.get_dbcog()
+    #     mgraph = dbcog.database.graph
+    #     final = []
+    #     for i in range(0, 8800):
```

Comment:
`graph.max_monster_id` holds the greatest monster ID if you want to use that instead of hardcoding it